model/src/main/java/com/epam/esm/repository/GiftCertificateRepository.java

```diff
@@ -0,0 +1,75 @@
+package com.epam.esm.repository;
+
+import com.epam.esm.entity.GiftCertificate;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Implementors of the interface provide functionality for manipulating stored {@link GiftCertificate} entities.
+ *
+ * @author Dmitry Karnyshov
+ */
+public interface GiftCertificateRepository {
```

Comment:
Let's put repository classes to the `service` layer or create a separate subproject called `repository` and put them there.

Reply:
Moved.